### PR TITLE
YONK-806: Organization view APIs for mobileapp resource

### PR DESCRIPTION
**Pull request by aamishbaloch**
Changes include view for organization view APIs for mobile app resource, through which organizations can be added or removed from the mobile apps. Unit tests are also included.

### CODE REVIEW

**Comment by aamishbaloch** [comment]
@ziafazal Can you review this?

File: mobileapps/tests.py [hunk 1]
```diff
@@ -383,3 +383,134 @@ def test_mobileapps_users_remove(self):
         self.assertEqual(response.data['count'], 2)
         self.assertEqual(len(response.data['results']), 2)
         self.assertEqual(response.data['num_pages'], 1)
+
+
+@ddt.ddt
+class MobileappsOrganizationApiTests(ModuleStoreTestCase, APIClientMixin):
+    """ Test suite for Mobileapps User API views """
```
Comment (ziafazal) [inline mobileapps/tests.py]: Typo: Mobileapps User API views 
should be organization API

File: mobileapps/tests.py [hunk 2]
```diff
@@ -383,3 +383,134 @@ def test_mobileapps_users_remove(self):
...
+        self.test_mobileapp_operating_system = 1
+        self.test_mobileapp_current_version = str(uuid.uuid4())
+        self.organizations = [
+            Organization.objects.create(name='ABC Organization'),
+            Organization.objects.create(name='XYZ Organization'),
+        ]
+
+        self.mobileapp = self._setup_test_mobileapp()
+
+        self.user = UserFactory.build(username='test', email='test@edx.org', password='test_password')
```
Comment (ziafazal) [inline mobileapps/tests.py]: this statement should be enough to create a user
```
UserFactory.create(username='test', email='test@edx.org', password='test_password')
```
we don't need to  call `save` explicitly.

File: mobileapps/tests.py [hunk 3]
```diff
@@ -383,3 +383,134 @@ def test_mobileapps_users_remove(self):
...
+        self.user.save()
+        self.client = Client()
+        self.client.login(username=self.user.username, password='test_password')
```
Comment (ziafazal) [inline mobileapps/tests.py]: why we need to login a user to these tests?

**Review by ziafazal — CHANGES_REQUESTED** [review]
@aamishbaloch completed first round of review.

File: mobileapps/tests.py (hunk 3 above)
Comment (aamishbaloch) [reply mobileapps/tests.py]: We are getting "updated_by" attribute from the logged in user. That's why I have to login the user here.

**Comment by aamishbaloch** [comment]
@ziafazal Done with the changes, Can you review?

**Review by ziafazal — APPROVED** [review]

**Pull request merged**